packages/modelserver-node/src/server-integration-test.spec.ts

```diff
@@ -278,6 +285,60 @@ describe('Server Integration Tests', () => {
                 });
             }
         });
+
+        it('Use default format (json-v2) for model update results', async () => {
+            const addPatch: Operation = {
+                op: 'add',
+                path: '/workflows/0/nodes',
+                value: {
+                    $type: 'http://www.eclipsesource.com/modelserver/example/coffeemodel#//ManualTask',
+                    name: 'New ManualTask'
+                }
+            };
+
+            const updateResult = await transaction.applyPatch(addPatch);
+            expect(updateResult).to.be.like({
+                success: true,
+                patch: [
+                    {
+                        op: 'add',
+                        path: '/workflows/0/nodes/-',
+                        value: {
+                            $type: 'http://www.eclipsesource.com/modelserver/example/coffeemodel#//ManualTask',
+                            $id: '//@workflows.0/@nodes.1',
+                            name: 'New ManualTask'
+                        }
+                    }
+                ]
+            });
+        });
+
+        it('Use json format (v1) for model update results', async () => {
+            const addPatch: Operation = {
+                op: 'add',
+                path: '/workflows/0/nodes',
+                value: {
+                    $type: 'http://www.eclipsesource.com/modelserver/example/coffeemodel#//ManualTask',
```

Comment:
This should be `eClass`, not `$type`?

But, then again, I don't see that anything is transforming the message from `json` format to `json-v2` when passing it along to the Java server, so perhaps I've missed some crucial point.